src/app/delegation_backend/src/sheets_test.go

```diff
@@ -0,0 +1,80 @@
+package delegation_backend
+
+import (
+	"encoding/hex"
+	"math/rand"
+	"reflect"
+	"testing"
+	"testing/quick"
+)
+
+func randRow(r *rand.Rand) ([](interface{}), *Pk) {
+	var v (interface{})
+	var pk *Pk
+	switch i := r.Intn(10); i {
+	case 0: // Random hexadecimal string
+		v = hex.EncodeToString(randBytes(r.Intn(100)+1, r))
+	case 1: // Random string starting with https://
+		v = "https://" + randString(r)
+	case 5: // Integer
```

Comment:
Nit: this ordering seems unusual, perhaps it makes sense to renumber these sequentially?